core-bundle/assets/styles/backend-search/container.pcss

```diff
@@ -1,10 +1,10 @@
-@media (max-width: 991.98px) {
+@media (width <= 991.98px) {
```

Comment:
Would it be safe to change this to `width < 992px`?

Reply:
Imho this should be safe.
We could also rewrite it to `width < 992px`  as the @media queries only allowed `equal to` and we can now use `less than` with the new range-media-query.

Could however use your suggestion as it's basically the same as before then.